CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+# dbt_snapchat_ads v0.9.0
+[PR #34](https://github.com/fivetran/dbt_snapchat_ads/pull/34) introduces the following updates:
+
+## Schema Updates
+
+**2 total changes • 1 breaking change**
+| **Model/Column** | **Change type** | **Old name** | **New name** | **Notes** |
+| ---------------- | --------------- | ------------ | ------------ | --------- |
+| `snapchat_ads__campaign_country_report` | New Transforms Model |   |   |  Uses `campaign_geo_country_daily_report` source table  |
+| `snapchat_ads__campaign_region_report` | New Transforms Model |   |   |  Uses `campaign_geo_region_daily_report` source table  |
+
+Please note that these are disabled by default. To enable them, add the following configuration to your root `dbt_project.yml` file:
```

Comment:
```suggestion
Please note that these are disabled by default for dbt Core users. To enable them, add the following configuration to your root `dbt_project.yml` file:
```
also could be good to link to the README section for more details

Reply:
updated